app/dashboard/templates/bounty/new.html

```diff
@@ -15,23 +17,15 @@
   along with this program. If not, see <http://www.gnu.org/licenses/>.
 {% endcomment %}
 {% load i18n static %}
-<!DOCTYPE html>
-<html lang="en">
 
-<head>
-  {% include 'shared/head.html' %}
+{% block head %}
   {% include 'shared/cards.html' %}
   <meta name="title" content="Create Funded Issue / Bounty | Gitcoin">
   <meta name="description" content="Instructions for creating an issue and posting a bounty to Gitcoin so developers across the world can discover your bounty and start working on your issue.">
   <link rel="stylesheet" href={% static "v2/css/submit_bounty.css" %}>
-</head>
+{% endblock head %}
 
-<body class="interior {{active}} g-font-muli" style="background-color: #FFFFFF;">
```

Comment:
Seems like the `style` attribute gone missing with this pr.
Or wasn't it used/or overruled anyway?

Reply:
it's not used. you can view https://gitcoin.co/funding/new and see that removing that doesn't change a thing